### PR TITLE
Milestone 2 - BookStore React

### PR DESCRIPTION
**Pull request by fivan18**
## Specific requirements
- [ ] Prepare the directory structure
- [ ] Implement the App component
- [ ] Implement the BooksList component
- [ ] Implement the Book component
- [ ] Implement the BooksForm component
- [ ] Define the Redux actions and reducers
- [ ] Initialize the Redux store

### CODE REVIEW

File: src/components/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (bellom) [inline src/components/BooksForm.js]: Good job so far!

we should keep `bookForm` inside a container folder
screenshot from pathwright below;
image

File: src/components/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (bellom) [inline src/components/BooksList.js]: Good job so far!

we should keep `bookList` inside a container folder
screenshot from pathwright below;
image

File: src/reducers/books.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    title: 'The everything store',
+    category: 'Biography',
+    id: Math.floor(Math.random() * 1000),
+  },
+  {
+    title: "The Hitchhiker's Guide to the Galaxy",
+    category: 'Sci-Fi',
+    id: Math.floor(Math.random() * 1000),
+  },
+];
```
Comment (bellom) [inline src/reducers/books.js]: According to the pathwright, you are supposed to create the book object inside `src/index.js`.
screenshot below;
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**1.`src/reducers/index.js` should combine all our reducers, but for now, we have only `book.js` reducer;
checkout [Combine Reducer](https://redux.js.org/recipes/structuring-reducers/using-combinereducers)**
screenshot below;
image

**2. Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by AdnanAfsari — APPROVED** [review]
**Status: Approved**

Hello @fivan18 @kazumaki ✋ 

Well done 👏 . It's a 👍  from me.

Happy Coding
Adnan Afsari

**Pull request merged**